### DISCUSSION TITLE
No additional context bar when selecting photos in Tags explorer

### DISCUSSION THREAD

**Discussion by Sargates** [immich-app/immich#17310] [Feature Request]
### The bug

There's no additional context bar when selecting multiple items in the Tags explorer like there is when selecting items on the Photos page or within an Album.

### From an Album:
![image](https://github.com/user-attachments/assets/58535335-10b6-43c2-8def-5c0ad9c9c60d)
### From "Tags" without selecting:
![image](https://github.com/user-attachments/assets/112fef54-1242-44cf-83b1-b38b74408a0d) 
### From "Tags" with pictures selected:
![image](https://github.com/user-attachments/assets/7a379f85-2c97-4a13-948c-504b14de63ba)

### The OS that Immich Server is running on

Debian 12

### Version of Immich Server

v1.18.2

### Version of Immich Mobile App

N/A

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

1. Make sure "Tags" are enabled in user settings
2. Tag multiple images
3. Go to "Tags"
4. Select images
5. There is no additional context bar like on the other pages. Unable to tag multiple items at a time in this page.

**Comment by bo0tzz** [comment]
#15022
